validation/src/static/styles/index.css

```diff
@@ -1,4 +1,334 @@
+  padding: 10px;
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.runComparisonInputSection {
+  display: flex;
+  flex-direction: column;
+  padding: 30px;
+}
+
+.heading {
+  font-size: 14pt;
+  color: #C9805C;
+  font-weight: bold;
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.bluetext {
+  font-size: 12pt;
+  color: #4A70D0;
+}
+
+.redtext {
+  font-size: 12pt;
+  color: #d04a4a;
+}
+
+.instructionTextContainer {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.runComparisonBtnContainer {
+  padding-top: 10px;
+  padding-bottom: 10px;
+  display: flex;
+}
+
+.runComparisonBtn {
+  margin-left: auto;
+}
+
+.comparisonSection {
+  margin-bottom: 20px;
+}
+
+#snippetInput {
+  width: 100%;
+}
+
+.tableHeaderRow {
+  background: #C9805C;
+  color: white;
+}
+
+table {
+  background: white;
+  border-collapse: collapse;
+  border: 1px solid black;
+  width: 100%;
+}
+
+td, th {
+  border: 1px solid black;
+  padding: 5px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.implTitle {
+  background: #C9805C;
+  color: white;
+  font-weight: bold;
+}
+
+#snippetHistoryContainer {
+  margin-top: 10px;
+  border: 1px solid black;
+  display: flex;
+  flex-direction: column;
+  padding: 5px;
+}
+
+#snippetHistoryContainer > a {
+  border: 1px solid black;
+  margin: 5px;
+}
+
+.linkBtn {
+  border: none;
+  background: #C9805C;
+  border-radius: 5px;
+  padding: 4px;
+  padding-left: 8px;
+  padding-right: 8px;
   color: white;
+  text-decoration: none;
+}
+
+.linkBtn:hover {
+  background: #8d593f;
+}
+
+.comparisonResultText {
+  padding-top: 10px;
+}
+
+.hidden {
+  display: none;
+}
+
+.visible {
+  display: block;
+}
+
+.urlComparisonContainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.urlInput {
+  font-size: 12pt;
+  padding: 10px;
+  font-family: 'Roboto Mono';
+  border: none;
+  border-radius: 5px;
+}
+
+#previewSection {
+  background: white;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+}
+
+.previewHeader {
+  display: flex;
+  justify-content: center;
+  background: #C9805C;
+  color: white;
+  font-size: 14pt;
+  font-weight: bold;
+  padding: 10px;
+  border-radius: 5px 5px 0px 0px;
+}
+
+.previewBody {
+  display: flex;
+}
+
+.comparisonType {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
+.comparisonTitle {
+  font-size: 12pt;
+  font-weight: bold;
+  color: #4A70D0;
+  padding: 10px;
+}
+
+.previewCardContainer {
+  display: flex;
+  flex-direction: column;
+  padding-left: 10px;
+  padding-right: 10px;
+  overflow: scroll;
+}
+
+.previewCard {
+  display: flex;
+  background: #C9805C;
+  color: white;
+  font-size: 12pt;
+  font-weight: bold;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  padding: 10px;
+}
+
+.previewCard > span {
+  margin-left: auto;
+}
+
+.previewCard:hover {
+  background: #8d593f;
+  cursor: pointer;
+}
+
+.invisibleLink {
+  text-decoration: none;
+}
+
+/* CSS for Loading Spinner from https://projects.lukehaas.me/css-loaders/ */
```

Comment:
IANAL but: When you're including third party code you should make sure you're obeying the license. In this case it's the MIT license https://github.com/lukehaas/css-loaders/blob/step2/LICENSE - which is good, we can use the code :) but it requires a notice.

I'd suggest moving all the 3rd party stuff to a separate file (`spinner.css`)  and start the file wiht

```
/** @license 
The MIT License (MIT)

Copyright (c) 2014 Luke Haas

Permission is hereby granted, free of charge, to any person obtaining a copy of
this software and associated documentation files (the "Software"), to deal in
the Software without restriction, including without limitation the rights to
use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
the Software, and to permit persons to whom the Software is furnished to do so,
subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
*/
```